cypress/integration/pages/frontPage/amp.js

```diff
@@ -21,6 +22,17 @@ Object.keys(config)
         });
       });
 
+      describe('ATI', () => {
+        it('should have an amp-analytics tag with the ati url', () => {
+          cy.hasAmpAnalyticsAtiUrl(
+            envConfig.atiUrl,
+            config[service].isWorldService
+              ? envConfig.atiAnalyticsWSBucket
+              : '',
+          );
+        });
+      });
```

Comment:
For completeness, we should have a test that checks for scripts on AMP pages too, e.g. A cypress command `cy.hasExpectedAmpScripts`.

Reply:
Likewise, front pages of both platforms should have a Chartbeat cypress test that asserts to not have the Chartbeat script.

Reply:
Agreed, not in scope for this PR though, its just moving stuff

Reply:
@dr3 can you open an issue for this, also referencing the TODO we added last night pls

Reply:
https://github.com/bbc/simorgh/issues/2948 :)